cloudinit/features.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+"""
+Feature flags are used as a way to easily toggle configuration
+**at build time**. They are provided to accommodate feature deprecation and
+downstream configuration changes.
+
+Currently used upstream values for feature flags are set in
+``cloudinit/features.py``. Overrides to these values (typically via quilt
+patch) can be placed
+in a file called ``feature_overrides.py`` in the same directory. Any value
+set in ``feature_overrides.py`` will override the original value set
+in ``features.py``.
+
+Each flag should include a short comment regarding the reason for
+the flag and intended lifetime.
+"""
```

Comment:
We went through quite a lot of discussion about testing of feature flags; I'd appreciate it if we could also include our conclusions on that in the documentation (so that we don't have to re-litigate it every few months :grin:).

Reply:
Added a sentence on testing. Let me know if you think more is needed.